packages/db-objects-file/lib/objects/objectsInMemFileDB.js

```diff
@@ -2006,21 +2006,11 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
             if (options.user) {
                 obj.acl.owner = options.user;
```

Comment:
here the overwrite has happened because options.user was always set to admin if not explicitly provided in setObj settings

Reply:
I would expect the same, yes. What we do when adapters create objects? Then also apply default rights? Most likely yes ... in fact the adapters themself can still do anything because Admin-like